server/webdriver/shared_tests/place_explorer_i18n_test.py

```diff
@@ -25,43 +28,69 @@ class PlaceI18nExplorerTestMixin():
   def test_japan_in_japanese(self):
     """Test translations from various sources are displayed correctly."""
 
-    start_url = self.url_ + '/place/country/JPN?hl=ja&disable_dev_places=true'
```

Comment:
should we fork these, so the old pages are tested until we complete the migration?

i think at least one test for i18n for the old pages should be kept